store.go

```diff
@@ -2320,24 +2320,53 @@ func (s *store) DeleteContainer(id string) error {
 
 	if rcstore.Exists(id) {
 		if container, err := rcstore.Get(id); err == nil {
+			var (
+				wg        sync.WaitGroup
+				rlError   error
+				rcError   error
+				rmGcError error
+				rmRcError error
+			)
```

Comment:
Can we replace the variables above with one error channel? I think that would be a bit more idiomatic and ultimately less lines of code. We can wrap the errors into one before returning.

Reply:
yes sure, fixed in the new version.